mirrord/operator/src/client.rs

```diff
@@ -137,10 +138,23 @@ impl OperatorApi {
         }
     }
 
+    /// Create websocket connection to operator
+    ///
+    /// Note: Uses `MIRRORD_OPERATOR_LAYER_SESSION` to have a persistant session_id across child
+    /// processes
     async fn connect_target(
         &self,
         target: TargetCrd,
     ) -> Result<(mpsc::Sender<ClientMessage>, mpsc::Receiver<DaemonMessage>)> {
+        let session_id: u64 = std::env::var(MIRRORD_OPERATOR_LAYER_SESSION)
+            .ok()
+            .and_then(|val| val.parse().ok())
```

Comment:
Shouldn't we differentiate between parse failing (invalid) and no ID exists (valid)?